contrib/exporters/docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+---
+version: '2.2'
+
+services:
+  zcashd:
+#    command: ["-reindex"]
+    image: electriccoinco/zcashd:latest
+#    image: localhost:5000/electriccoinco/zcashd:v5.0.0
+    mem_limit: 8G
+    volumes:
+      - $ZCASHD_DATADIR:/srv/zcashd/.zcash
+      - $ZCASHD_PARMDIR:/srv/zcashd/.zcash-params
+    env_file:
+      - .env
+    ports:
+      - "$ZCASHD_RPCPORT:$ZCASHD_RPCPORT"
+      - "$ZCASHD_PROMETHEUSPORT:$ZCASHD_PROMETHEUSPORT"
+    cpus: 2
+    logging:
+      driver: loki
+      options:
+        loki-url: 'http://localhost:3100/api/prom/push'
+
+  ecc_zcashd_exporter:
+    build: .
+    env_file:
+      - .env
+    ports:
+      - "9200:9200"
+    logging:
+      driver: loki
+      options:
+        loki-url: 'http://localhost:3100/api/prom/push'
+
+  node_exporter:
+    image: prom/node-exporter:latest
+    container_name: mon_node_exporter
+    restart: unless-stopped
+    expose:
+      - 9100
+
+  grafana:
+    #image: grafana/grafana:8.2.2
+    image: grafana/grafana
+    ports:
+      - "3000:3000"
+    env_file:
+      - .env
+    volumes:
+      - ./docker/grafana/provisioning/:/etc/grafana/provisioning/
```

Comment:
This is nice, I'll add the dashboard we have for the metrics that zcashd exposes to #6176